istioctl/cmd/tag.go

```diff
@@ -41,6 +43,7 @@ const (
 	defaultRevisionName         = "default"
 	pilotDiscoveryChart         = "istio-control/istio-discovery"
 	revisionTagTemplateName     = "revision-tags.yaml"
+	minRevisionTagIstioVersion  = "1.10"
```

Comment:
Can you use the version number from the following? Good to avoid hardcoding versions at many places.
 
```suggestion
	minRevisionTagIstioVersion  = version.OperatorCodeBaseVersion
```

Reply:
If not please add a comment above the `minRevisionTagIstioVersion` why it's needed.

Reply:
Right, revision tags depend on [this commit](https://github.com/istio/istio/pull/29583) which is first in 1.10, added a comment to that effect :+1: